controllers/internal/utils/annotations_test.go

```diff
@@ -0,0 +1,76 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var a2 = map[string]string{"k8gb.io/primary-geotag": "270", "k8gb.io/strategy": "failover"}
```

Comment:
Consider using geotag names for broader community:
```suggestion
var a2 = map[string]string{"k8gb.io/primary-geotag": "eu", "k8gb.io/strategy": "failover"}
```

Reply:
@somaritane Good idea